homeassistant/components/geniushub/switch.py

```diff
@@ -70,7 +70,10 @@ def is_on(self) -> bool:
 
         The zone is considered 'on' if & only if it is override/on (e.g. timer/on is 'off').
```

Comment:
Should we update the docstring as it sounds like it's contradicting the code now?

Reply:
Oh, good point! @GeoffAtHome Care to do the honours?

Reply:
Yep.